certbot-dns-google/certbot_dns_google/dns_google_test.py

```diff
@@ -76,7 +76,7 @@ def _setUp_client_with_mock(self, zone_request_side_effect):
 
     @mock.patch('oauth2client.service_account.ServiceAccountCredentials.from_json_keyfile_name')
     @mock.patch('certbot_dns_google.dns_google.open',
-                mock.mock_open(read_data='{"project_id": "' + PROJECT_ID + '"}'))
+                mock.mock_open(read_data='{"project_id": "' + PROJECT_ID + '"}'), create=True)
```

Comment:
Without this change, tests fail like this https://travis-ci.org/certbot/certbot/jobs/245125937#L5394 due to the older version of `mock`. Rather than just patching `open()` globally, we can use `create=True` to have `mock.patch()` create an `open` attribute on the specific module we're testing.